### PR TITLE
TizenRT: skip tests to run without hung and coredump

### PR DESCRIPTION
**Pull request by kolipka**
+ test_spi.js file was added in order to comply with testsets.json

IoT.js-DCO-1.0-Signed-off-by: Konrad Lipner k.lipner@samsung.com

### CODE REVIEW

**Comment by kolipka** [comment]
as a follow up I will add commit that change "skip": ["all"] to "skip": ["linux"] for all PerhiperialIO tests. Along with those tests. If module like ADC is not compiled in iotjs test will not be performed anyway, no need to skip tests like this. If it is included it should be tested.
To have it properly, I will change hardware tests to check as much as I can without need for external device, just as it is in SPI case here.

**Comment by yichoi** [comment]
```
Detected style problems:
/home/travis/build/Samsung/iotjs/test/run_pass/test_spi.js:51: trailing whitespace
* total lines of code: 34516
* total non-blank lines of code: 27422
* style errors: 1
* clang-format errors: 0
* total errors: 1
Failed tidy check
```

**Comment by yichoi** [comment]
It's not really fix about hung and coredump just skip tests. I would like to resolve it properly or I will close it

**Comment by kolipka** [comment]
@yichoi 
Since TizenRT is not mature yet, there is no easy way to workaround core dumps. It will take unknown amount of time. This fix, skipping crashing tests, enable running full test suite, and tracking progress of improvements.

**Comment by yichoi** [comment]
if so, rename the title of this PR properly.

**Comment by kolipka** [comment]
@yichoi Done

**Comment by LaszloLango** [comment]
The commit message is still incorrect.

**Comment by kolipka** [comment]
@LaszloLango why?
Before commit, testsets.json point to non existing spi.js file. This is bug and it is fixed in this commit.

**Comment by LaszloLango** [comment]
It is misleading like the title was before. The patch does not fix any test, but only skip them. Moreover please write full sentences.

**Comment by kolipka** [comment]
@LaszloLango 

https://github.com/Samsung/iotjs/blob/master/test/testsets.json#L89 points to test_spi.js file, this file is not present in our repository. This is bug. In this case I am not skipping test. I've added new file: https://github.com/Samsung/iotjs/pull/1065/files#diff-e2477c4d3df8a43cfdc91bc1fe235209R1

As you wish, I will enhance it to full sentence.

File: test/testsets.json [hunk 1]
```diff
@@ -16,64 +16,64 @@
     { "name": "test_dgram_multicast_set_multicast_loop.js", "skip": ["all"], "reason": "need to setup test environment" },
     { "name": "test_dgram_setttl_client.js", "skip": ["all"], "reason": "need to setup test environment" },
     { "name": "test_dgram_setttl_server.js", "skip": ["all"], "reason": "need to setup test environment" },
-    { "name": "test_dns.js" },
-    { "name": "test_dns_lookup.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_dns.js", "skip": ["tizenrt"], "reason": "hung" },
+    { "name": "test_dns_lookup.js", "skip": ["nuttx", "tizenrt"], "reason": "not implemented for nuttx" },
```
Comment (yichoi) [inline test/testsets.json]: I would better to add the reason why this test is skipped for tizenRT.

File: test/testsets.json [hunk 2]
```diff
@@ -16,64 +16,64 @@
...
-    { "name": "test_fs_writefile.js" },
-    { "name": "test_fs_writefile_sync.js" },
-    { "name": "test_fs_writefile_unlink.js" },
-    { "name": "test_fs_writefile_unlink_sync.js" },
-    { "name": "test_fs_event.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_fs_open_read.js" },
-    { "name": "test_fs_open_read_sync_1.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_fs_open_read_sync_2.js" },
-    { "name": "test_fs_open_read_sync_3.js" },
+    { "name": "test_fs_fstat.js", "skip": ["nuttx", "tizenrt"], "reason": "not implemented for nuttx" },
```
Comment (yichoi) [inline test/testsets.json]: ditto and so forth

**Comment by kolipka** [comment]
since it was processed so long, I've already made new one #1080 . SPI test fix will be applied later on.

**Pull request closed without merging**